devito/ops/types.py

```diff
@@ -112,6 +112,9 @@ def as_coeff_Mul(self):
     def as_coeff_Add(self):
         return sympy.S.Zero, self
 
+    def as_ordered_factors(self):
+        return (self,)
```

Comment:
Note for review.
This is necessary for `_print_Mul` because they define `_as_coeff_Mul()` above.